object_detector:
  ros__parameters:
    marker_length: 0.03
    cluster_radius: 0.2
    blue_id: 36
    yellow_id: 47
